functions-and-operators/tidb-functions.md

```diff
@@ -0,0 +1,190 @@
+---
+title: TABLE | TiDB SQL Function Reference
+summary: An overview of the usage of TiDB specific functions for the TiDB database.
+---
+
+# TIDB_BOUNDED_STALENESS
+
+The `TIDB_BOUNDED_STALENESS` function is an internal function of TiDB.
+
+# TIDB_DECODE_KEY
```

Comment:
I don't think these should all be H1s. The H1 should be "TiDB Specific Functions" and these should be H2s underneath.

Reply:
I changed the H1s to H2s and H2s to H3s.